skl2onnx/common/_container.py

```diff
@@ -267,26 +275,56 @@ def add_initializer(self, name, onnx_type, shape, content, can_cast=True):
             to cast the constant
         :return: created tensor
         """
-        if (can_cast and isinstance(content, np.ndarray) and
+        if (can_cast and isinstance(content, (np.ndarray, coo_matrix)) and
                 onnx_type in (TensorProto.FLOAT, TensorProto.DOUBLE) and
                 onnx_type != self.proto_dtype):
             content = content.astype(self.dtype)
             onnx_type = self.proto_dtype
 
+        sparse_tensor = None
+        tensor = None
+
         if isinstance(content, TensorProto):
             tensor = TensorProto()
             tensor.data_type = content.data_type
             tensor.name = name
             tensor.raw_data = content.raw_data
             tensor.dims.extend(content.dims)
+        elif (SparseTensorProto is not None and
+                isinstance(content, SparseTensorProto)):
+            raise NotImplementedError("Not implemented yet.")
```

Comment:
Should we also mention the link to file github issue like in other places?

Reply:
You're right.